events/producer/events.go

```diff
@@ -288,10 +285,11 @@ func registerHandler(ie *pb.Interest, h *handler) error {
 }
 
 func deRegisterHandler(ie *pb.Interest, h *handler) error {
-	producerLogger.Debugf("deRegisterHandler %s", ie.EventType)
+	//producerLogger.Debugf("deRegisterHandler %s", ie.EventType)
```

Comment:
Yea, I commented it out intentionally as it was getting displayed during the unit tests. I can add back if you would like or just remove.